common/login_tuple.cc

```diff
@@ -148,7 +148,7 @@ void LoginTuple::setDeviceAttrs(const json11::Json& msg, const std::shared_ptr<U
       device_attrs.insert(std::make_pair("app.minor", oxmad.app.minor));
       device_attrs.insert(std::make_pair("device.family", oxmad.device.family));
     }
-    dcache.addToCache(my_device_id, device_attrs);
+    dcache.addToCache(my_device_id, std::move(device_attrs));
```

Comment:
We can't use std::move here - device_attrs is a member variable and this will leave it undefined, and it's needed. That's why the test is failing